import_data_into_db.py

```diff
@@ -0,0 +1,64 @@
+import pandas as pd
+from sqlalchemy import Column, Float, Integer, String, create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+Base = declarative_base()
+
+
+class Animes(Base):
+    __tablename__ = "animes"
```

Comment:
I would add an empty line after class definition line.

Reply:
Line added in commit #824e425